### PR TITLE
Write /boot/cpuid.txt

### PR DESCRIPTION
**Pull request by Ealdwulf**
This PR replaces https://github.com/KanoComputing/kano-toolset/pull/263
 Moved from kano-network-hook to avoid dependency on network.

### CODE REVIEW

File: bin/kano-settings-onboot [hunk 1]
```diff
@@ -125,6 +129,25 @@ def main(args):
 
     return rc or RC_SUCCESS
 
+def write_cpu_id():
+    # Get the unit CPU Serial number and send it, save a local copy
+    cpuid = get_cpu_id()
+    cpuid_filename_boot = '/boot/cpuid.txt'.format(cpuid)
```
Comment (radujipa) [inline bin/kano-settings-onboot]: Missing `{}` somewhere here?

Comment (Ealdwulf) [reply bin/kano-settings-onboot]: No, we don't want the cpuid in the filename, unlike the one in `/etc`. That's a no-op, I'll remove it.

Comment (skarbat) [reply bin/kano-settings-onboot]: I think we want to remove the format(), to keep a fixed filename?

Comment (Ealdwulf) [reply bin/kano-settings-onboot]: Fixed.

**Pull request merged**